fedmsg_meta_umb/mbs.py

```diff
@@ -40,10 +40,8 @@ def subtitle(self, msg, **config):
         return tmpl.format(**msg['msg'])
 
     def link(self, msg, **config):
-        link = msg['msg']['state_url']
-        if not link:
-            return None
-        return link.replace('http://localhost', 'https://mbs.engineering.redhat.com')
+        idx = msg['msg']['id']
+        return 'https://mbsweb.engineering.redhat.com/module/%i' % idx
```

Comment:
Unfortunately, we don't have `https` enabled on this service yet. It's on my TODO list.

Reply:
All good.  I removed https for now.  :)